device/iot-device-samples/send-event/src/main/java/samples/com/microsoft/azure/sdk/iot/SendEvent.java

```diff
@@ -68,14 +68,14 @@ else if (status == IotHubConnectionStatus.CONNECTED)
     }
 
     /**
-     * Sends a number of messages to an IoT Hub. Default protocol is to 
+     * Sends a number of messages to an IoT or Edge Hub. Default protocol is to
      * use MQTT transport.
      *
      * @param args
-     * args[0] = IoT Hub connection string
-     * args[1] = number of requests to send
+     * args[0] = IoT Hub or Edge Hub connection string
+     * args[1] = number of messages to send
      * args[2] = protocol (optional, one of 'mqtt' or 'amqps' or 'https' or 'amqps_ws')
-     * args[3] = path to certificate to enable one-way authentication over ssl for amqps (optional, default shall be used if unspecified).
+     * args[3] = path to certificate to enable one-way authentication over ssl for amqps (optional, when connecting to IoT Hub, required if connecting to an Edge device using a non public root CA certificate).
      */
```

Comment:
"Optional when connecting to IotHub" is a little misleading. We shouldn't encourage users to provide special certificates just to connect to IotHub since the default certs work just fine.

how about "Not necessary when connecting directly to Iot Hub, but required if connecting to an Edge device using a non public root CA certificate"

Reply:
Will fix.